QUARKUS-343.md

```diff
@@ -0,0 +1,48 @@
+# QUARKUS-343 - Quarkus and non JSON payload in REST endpoints
+
+## Scope of the testing
+The test scope includes validating Quarkus's capability to handle non-JSON payloads (XML, YAML, and images) efficiently in REST endpoints.
+Tests will cover both JVM and Native mode executions.
+#### Test scenarios
+
+        1. XML Payload Scenario:
+            * Verify the endpoint's ability to consume and produce XML payloads.
+            * Send a request with an XML payload to the endpoint.
+            * Validate the response, ensuring it matches the expected XML structure.
+    
+        2. YAML Payload Scenario:
+            * Check the endpoint's ability to return YAML from a YAML file.
+            * Send a request with a YAML payload from a file to an endpoint.
+            * Validate the response, ensuring it matches the expected YAML content.
+    
+        3. Image Payload Scenario:
+            * Verify the endpoint's ability to return an image by making a GET request.
+            * Send an image in a multipart form to the endpoint.
+            * Check the endpoint's handling of image upload as part of a multipart form by making a POST request.
+### Impact on testsuites and testing automation:
+New tests will be added inside [http/http-advance-reactive](://github.com/quarkus-qe/quarkus-test-suite/tree/main/http/rest-client-reactive(http/http-advance-reactive)) module.
+
+### Impact on resources:
+There will be longer tests execution for jvm and native runs of QE TS.
+
+## Getting familiar with the feature
+
+Following actions were taken to ensure familiarity:
+- Experiment with Quarkus REST functionalities for providing and consuming non JSON payload (XML, YAML, images)
```

Comment:
```suggestion
- Experiment with Quarkus REST functionalities for providing and consuming non-JSON payload (XML, YAML, images)
```

Reply:
You keep repeating non-JSON payloed with XML / YAML / images, but that's not all. I'd suggest to add one simple sentence explaining why you chose these 3 somewhere in the TP.